release/serve_tests/workloads/autoscaling_single_deployment.py

```diff
@@ -173,7 +171,7 @@ def main(
     logger.info(f"Starting wrk trial on all nodes for {trial_length} ....\n")
     # For detailed discussion, see https://github.com/wg/wrk/issues/205
     # TODO:(jiaodong) What's the best number to use here ?
-    all_endpoints = list(_private_api.list_deployments().keys() - {"proxy"})
+    all_endpoints = ["/echo"]
```

Comment:
Why is this hard coded? We don't need the logic from before?

Reply:
This test is only deploying the `Proxy` deployment and `Echo` deployment (in their own applications). Not sure why we had this original logic, but setting it to just the route prefix for the `Echo` deployment should be equivalent.

Reply:
Got it, thanks!